test/integ/aio/test_file_transfer_agent_performance.py

```diff
@@ -0,0 +1,179 @@
+#
```

Comment:
I feel perf test should not be part of the integ test which focuses on the correctness of functionalities / not run against every commit but run daily.

what do you think?

Reply:
I removed it from the test folder